docker-compose.yml

```diff
@@ -35,6 +35,7 @@ x-search-api-env: &search-api-env
   RABBITMQ_PASSWORD: guest
   RACK_ENV: production
   REDIS_URL: redis://redis
+  RAILS_LOG_TO_STDOUT: "false"
```

Comment:
This environment variable looks a bit fishy because Search API isn't a Rails app. I guess this is one we've added ourselves?